src/common/containers/build-details.js

```diff
@@ -26,6 +27,9 @@ class BuildDetails extends Component {
         { this.props.isFetching &&
           <span>Loading...</span>
         }
+        { this.props.error &&
+          <Message status='error'>{ this.props.error.message || this.props.error }</Message>
```

Comment:
The PropTypes validation states `error` can be a bool. What is shown here when it is?